lib/sentry/logger_handler.ex

```diff
@@ -341,18 +341,11 @@ defmodule Sentry.LoggerHandler do
   # "report" here is of type logger:report/0, which is a map or keyword list.
   defp log_unfiltered(%{msg: {:report, report}}, sentry_opts, %__MODULE__{} = config) do
     case Map.new(report) do
-      %{report: %{reason: {exception, stacktrace}}}
+      %{reason: {exception, stacktrace}}
       when is_exception(exception) and is_list(stacktrace) ->
         sentry_opts = Keyword.merge(sentry_opts, stacktrace: stacktrace, handled: false)
         capture(:exception, exception, sentry_opts, config)
 
-      %{report: %{reason: {reason, stacktrace}}} when is_list(stacktrace) ->
-        sentry_opts = Keyword.put(sentry_opts, :stacktrace, stacktrace)
-        capture(:message, "** (stop) " <> Exception.format_exit(reason), sentry_opts, config)
-
-      %{report: report_info} ->
-        capture(:message, inspect(report_info), sentry_opts, config)
```

Comment:
We were never hitting these clauses? Did `mix coveralls.html` confirm that? 😮

Reply:
I think so? I mean, the whole function head can be deleted and tests still pass...
![CleanShot 2024-08-07 at 08 57 11](https://github.com/user-attachments/assets/bbd6cdbc-3ee0-4f46-af91-6b5d8ae9db7d)

Reply:
Uh, interesting. Well, not good 😄 I suspect that maybe some of those were for older Erlang versions but your change passes on all supported Erlang/Elixir versions, so we're good to go!